src/components/Popups/PopupDrawingToolbar.scss

```diff
@@ -22,64 +57,20 @@
     display: flex;
     align-items: center;
     justify-content: center;
-
-    &::after {
-        position: absolute;
-        right: 0;
-        width: 1px;
-        height: 100%;
-        background-color: #000;
-        content: '';
-    }
+    border-right: 1px solid #000;
 }
 
-.ba-PopupDrawingToolbar-grabber {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    margin: 8px;
-    cursor: move;
-}
-
-.ba-PopupDrawingToolbar-comment,
 .ba-PopupDrawingToolbar-delete,
 .ba-PopupDrawingToolbar-redo,
 .ba-PopupDrawingToolbar-undo {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    padding: 6px 8px;
-    color: $white;
-    background: transparent;
-    border: none;
-    border-radius: $bdl-border-radius-size;
-    outline: none;
-    cursor: pointer;
-    opacity: .7;
-    transition: opacity 200ms ease;
-
-    &:focus,
-    &:hover {
-        opacity: 1;
-    }
-
-    &[disabled] {
-        opacity: .2;
-    }
-}
-
-.ba-PopupDrawingToolbar-delete,
-.ba-PopupDrawingToolbar-redo,
-.ba-PopupDrawingToolbar-undo {
-    path {
-        fill: #fff;
-    }
+    @include PopupDrawingToolbarIconButton;
 }
 
 .ba-PopupDrawingToolbar-comment {
+    @include PopupDrawingToolbarButton;
     @include common-typography;
 
-    padding: 6px 18px;
+    padding: 0 18px;
```

Comment:
Should the default padding be reset in the mixin?

Reply:
Do we need to re-set top/bottom padding or can we use `padding-left` and `padding-right`? It's usually a good convention to override only the minimum set of styles.